### PR TITLE
Update primary color and styles

### LINKED ISSUE #1549: Change primary color and update style
- [ ] Change primary color to: `#1eacf8`
- [ ] Create secondary color : `#0274dc`
- [ ] Use secondary color on buttons follow instead of the primary color
- [ ] Change topic style to: `color: #777676; background-color: #e9e7e7;` and remove borders

### PR DESCRIPTION
**Pull request by jm90m**
Fixes #1549.

Changes:
- [x] Change primary color to: `#2088ff`
- [x] Create secondary color : `#79b8ff`
- [x] Use secondary color on buttons follow instead of the primary color
- [x] Change topic style to: `color: #777676; background-color: #e9e7e7;` and remove borders

- Also remove all unused colors and update colors throughout site to use secondary variable

### CODE REVIEW

**Comment by bonustrack** [comment]
@pareshUD we need a new image without the purple for the Hero can you work this out?

**Comment by bonustrack** [comment]
Can you use the secondary color for the hero background @jm90m ?

**Comment by pareshUD** [comment]
@bonustrack On it I will update that in a while

**Comment by pareshUD** [comment]
1. Illustration is going on left when the page is loading and then get in place.
screen shot 2018-04-03 at 2 32 37 pm

2. Can we make the illustration bit bigger if possible?

3. We need to change the color of last illustration (Steem coin should have same color for all lines)

4. Remove circles from all illustrations.

**Comment by bonustrack** [comment]
@pareshUD We have some issue making the slider working well when loading. Can we have a proposal with the 3 illustrations at once? (without the slides)

**Comment by pareshUD** [comment]
@bonustrack  Check this out

busy_banner
busy_banner2

**Comment by pareshUD** [comment]
final banner

Check this @f

**Comment by bonustrack** [comment]
Can you make a proposal for mobile view @pareshUD ?

**Comment by bonustrack** [comment]
The integration is not exactly like what @pareshUD did
- [ ] The bullet number text should be bold
- [ ] Bullet circle should be smaller
- [ ] Space between bullet and title should be bigger
- [ ] Illustration should be smaller and centered
- [ ] Background should be same before (without cover and resizing)

Don't work on the slider yet, we are waiting for @pareshUD proposal for mobile

**Comment by bonustrack** [comment]
Can you move illustrations and title to 2 lines. First line (1 in image) with the number of the step and the title, same style than on desktop.
2: Illustration centered on new line
3: Slider buttons
image
Make sure the spacing in top and bottom are same size and at least 20px

**Comment by bonustrack** [comment]
Can you resolve conflicts @jm90m ?

File: assets/images/busy.svg [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```
Comment (bonustrack) [inline assets/images/busy.svg]: Can you remove this file since we dont use it on this PR anymore?

**Comment by bonustrack** [comment]
@Sekhmet can you review this PR?

**Comment by Sekhmet** [comment]
I think I fixed the issue with centering slider on mobile, so you might want to take a second look at it to make sure it works properly.

**Comment by bonustrack** [comment]
There is an issue with scrollbar, the space with green line is hidden under scrollbar. Right part of the image not appear because of this and there is not space between reward and right side.
image

**Comment by jm90m** [comment]
@bonustrack can you show me this? I can't reproduce

**Pull request merged**